src/sentry/ingest/transaction_clusterer/rule_validator.py

```diff
@@ -52,3 +57,20 @@ def _is_schema_and_all_stars(self) -> bool:
             return False
 
         return self._is_string_all_stars(url.path)
+
+    def _is_http_method_and_all_stars(self) -> bool:
+        """
+        Return true if the rule looks like an HTTP method and stars.
+
+        ## Examples
+        `GET /*/*/**` -> `True`
+        `GET /a/*/**` -> `False`
+
+        Transaction names with this pattern are commonly sent by SDKs and like
+        the all stars case, rules of this form provide no value.
+        """
+        match = re.match("[a-z]+ (.*)", self._rule, re.IGNORECASE)
```

Comment:
Maybe slightly surprising that the match group is `[a-z]` rather than `(GET|POST|PATCH|PUT|DELETE)` but I couldn't think of any reason why it'd be a problem!

Reply:
We could strip `(GET|POST|PATCH|PUT|DELETE) ` from the beginning of the rule before calling `is_valid` (or right at the start of `is_valid`), then we don't need a third method and we would also catch cases like `GET https://*/**`.